src/components/styles/modal.less

```diff
@@ -33,11 +33,14 @@
         &.large {
             width: 60%;
             max-width: 1000px;
+
+            @media @small_screen {
```

Comment:
@henningmu here I've added a dedicated `small_screen` media query for large. this way we avoid the override. 👍